src/components/TopBar/TopBarContent.tsx

```diff
@@ -74,13 +94,24 @@ export default function TopBarContent(props: TopBarProps): JSX.Element | null {
             <Svg.Logo className={classes.logo} />
           </Link>
         </div>
+
         {organizations && organizations.length > 0 && (
           <>
             <div className={classes.separator} />
+            {isAcceleratorRoute && user && isAcceleratorAdmin(user) && (
+              <div>
+                <Link fontSize={16} to={APP_PATHS.HOME}>
+                  Terraware
+                </Link>
+                <span className={classes.breadcrumbSeparator}>/</span>
+                <span className={classes.breadcrumbSelected}>{strings.ACCELERATOR_CONSOLE}</span>
+              </div>
+            )}
```

Comment:
This was small enough I wasn't sure if we'd prefer to wrap this up as a component or leave as-is, happy to component-ify if desired.